packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/00-page-header.json

```diff
@@ -3,41 +3,15 @@
     "url": "#",
     "links": [
       {
-        "text": "Lorem ipsum",
+        "text": "Stories",
         "url": "#"
       },
       {
-        "text": "Quis autem",
+        "text": "People",
         "url": "#"
       },
       {
-        "text": "Sed ut perspiciatis unde omnis",
-        "url": "#"
-      },
-      {
-        "text": "Neque porro",
-        "url": false
-      },
-      {
-        "text": "Nemo",
-        "url": "#"
-      },
-      {
-        "text": "Unquam sit",
-        "url": "#"
-      },
-      {
-        "text": "Mors tua vita mea",
-        "url": "#"
-      }
-    ],
-    "externalLinks": [
-      {
-        "text": "Praesent eu",
-        "url": "#"
-      },
-      {
-        "text": "Donec sagittis",
+        "text": "Jobs",
```

Comment:
Agreed. In fact, I'd actually suggest reverting this file.

For the "atoms" through to the "templates", we tend to use dummy content and values. Partly it's not to give the false impression that a component is intended for a particular site or app. But also, it's useful to stress-test things a little. In this case, we intentionally had an unusually large number of links, so that we can see if the header's layout still holds up.

The "pages" category, is where we populate things with realistic content to provide a sort of sample of what you can build with Gravity.

So yeah, I'd suggest reverting this file. But making the equivalent changes to the various pages' JSON files. Currently, they all have the "About Us, Careers, Locations and Blog" links, so you could perhaps change those to "Stories, People and Jobs" as per the new site design.